jmh/benchmark-binary-tree/README.md

```diff
@@ -0,0 +1,98 @@
+# Binary Tree Benchmark 
+
+This demo shows how to run a JMH benchmark as a native image.
+
+In order to build a native image of this application we need to run the native image tracing agent to supply
+the reflection config to the native iameg builder. This has already been done for you to save time and the generated 
+configuration can be found in, `src/main/resources/META-INF/native-image/`.
+
+> **NOTE** To generate the configuration yourself, you will need to ensure that the JMH `fork` parameter is set to 0,
+> which can be done from the command line with, `-f 0`, and this can also be done within the code using the `@Fork` annotation.
```

Comment:
```suggestion
> which can be performed from the command line using the option  `-f 0`. It can also be achieved within the code by using the `@Fork` annotation.
```